pkg/systemd/services/services.jsx

```diff
@@ -545,8 +545,8 @@ class ServicesPage extends React.Component {
         } else if (unit.UnitFileState && unit.UnitFileState.indexOf('static') == 0) {
             unit.AutomaticStartup = _("Static");
             unit.AutomaticStartupKey = 'static';
-        } else if (unit.UnitFileState && unit.UnitFileState.indexOf('masked') == 0) {
-            unit.AutomaticStartup = _("Masked");
```

Comment:
This needs a follow-up anyway, so not such a big deal: But was there a particular reason to remove this case?